concrete/src/Entity/Page/Feed.php

```diff
@@ -349,18 +351,16 @@ public function getPageListObject()
             $pl->ignorePermissions();
         } else {
             $vp = \Concrete\Core\Permission\Key\Key::getByHandle('view_page');
-            $guest = \Group::getByID(GUEST_GROUP_ID);
-            $access = GroupEntity::getOrCreate($guest);
-            // we set page permissions to be Guest group only, because
-            // authentication won't work with RSS feeds
-            $pl->setPermissionsChecker(function ($page) use ($vp, $access) {
+            $user = new User();
```

Comment:
@biplobice just a note: use `$app->make(User::class)` instead of `new User()`.
With concrete5 8.5 it doesn't change anything, but with concrete5 8.6 it will be much faster.

Reply:
Thank you. Forgot about that.